app/util/weapon.py

```diff
@@ -0,0 +1,3 @@
+def get_weapon(knight: dict) -> dict:
+    knight["power"] += knight["weapon"]["power"]
+    return knight["power"]
```

Comment:
The same, no need ti change input config
It so bad practice to change input parameters